arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedParquetDefinitionLevelReader.java

```diff
@@ -305,7 +305,7 @@ public void nextDictEncodedBatch(
     protected abstract void nextVal(
         FieldVector vector, int idx, ValuesAsBytesReader valuesReader, int typeWidth, byte[] byteArray);
     protected abstract void nextDictEncodedVal(
-        FieldVector vector, int startOffset, VectorizedDictionaryEncodedParquetValuesReader reader,
+        FieldVector vector, int idx, VectorizedDictionaryEncodedParquetValuesReader reader,
```

Comment:
Do you know why this is needed now? I thought we already caught variable names that shadow field names.

Reply:
this is due to `ConsistentOverrides` from the additional ErrorProne checks in https://github.com/palantir/gradle-baseline#baseline-error-prone-checks